### ISSUE TITLE
gamecontrollerd (/usr/libexec/gamecontrollerd) triggers ReiKey

### ISSUE THREAD

**Issue by h-spiess** [objective-see/ReiKey#14]
Hey,

First of all, thank you for all your nice work on MacOS security. I really appreciate that.

Since updating to the newest version of MacOS 11.3, ReiKey (v.1.4.1) complains about _gamecontrollerd_ _(/usr/libexec/gamecontrollerd)_. After googling it seems that _gamecontrollerd_ is from the OS itself and should probably be ignored if the setting _"Ignore Apple Programs"_ is set. Correct me if i'm wrong. For now I disabled alerts manually.

Thanks :)

**Comment by jawiv** [comment]
Yes. I'm getting gamecontrollerd alerts every minute or so. Takes focus away from whatever program I'm typing in.

**Comment by marcel-bluestone** [comment]
I experience the same thing and it is really really annoying, as typing this text alone had me regain focus on the window text input-form three times.
It seems gamecontrollerd is not considered to be an Apple process.

**Comment by proton1k** [comment]
I've encountered the same issue. I found that gamecontrollerd for many users was never launched prior update of MacOS to Big Sur. I wonder how it might be related. I also have doubts regarding the Game Controller daemon lanched here and there just because of Chrome or an Electronjs. Yet here's the tricky question: if it's not really needed and the user never plays games, can it be shut down forever without troubled consequences for the OS?

If gamecontrollerd is passively listening to the keyboard tapping and might pass it to some 3rd-party app, imagine it's a web browser with a 3rd-party Javascript loaded, can a script request the information from this daemon or not?

**Comment by egrueter-dev** [comment]
I am having `usr/libexec/relatived` come up as a keylogger with passive listening permissions. Is this malware?

**Comment by utkonos** [comment]
In the ReiKey preferences, there is a setting `Ignore Apple Programs`. For me, this has removed `gamecontrollerd` from the listing when a scan is performed. I have not seen the annoying alert either since I make this change to preferences.

<img width="712" alt="prefs" src="https://user-images.githubusercontent.com/2775739/167261911-58048582-0a99-4e1a-a256-2b0de46d25fb.png">

**Comment by Lorentiii** [comment]
> I am having `usr/libexec/relatived` come up as a keylogger with passive listening permissions. Is this malware?

Did you ever find out what it actually was?
